web/src/message_view_header.ts

```diff
@@ -181,3 +181,11 @@ export function maybe_rerender_title_area_for_stream(modified_sub: StreamSubscri
         render_title_area();
     }
 }
+
+// same as above but accepts only stream ID
+export function maybe_rerender_title_area_for_stream_by_id(modified_sub_id: number): void {
```

Comment:
There is no need for a separate function, since the existing function also uses only stream ID. Would be good to just modify that to accept ID in a prepartory commit.

Reply:
ok